File: AIS/AIS.BLL/Models/QuestionIntervieweeAnswer.cs
namespace AIS.BLL.Models
{
    public class QuestionIntervieweeAnswer
    {
        public int Id { get; set; }
        public string Text { get; set; }
        public int Mark { get; set; }
        public int QuestionId { get; set; }
        public Question Question { get; set; }
        public int? SessionId { get; set; }
        public Session Session { get; set; }
    }
}

File: AIS.front/ais.front/src/core/interfaces/questionIntervieweeAnswer/questionIntervieweeAnswer.ts
import { ShortQuestion } from '../question/shortQuestion';
import { ShortSession } from '../session/shortSession';

export interface QuestionIntervieweeAnswer {
  id: number;
  text: string;
  mark: number;
  sessionId: number;
  session: ShortSession | null;
  questionId: number;
  question: ShortQuestion | null;
}
